src/TweetPublisher/TwitterApi/TwitterApiWrapper.php

```diff
@@ -75,6 +82,22 @@ public function publishTweet(string $status): void
         ]);
     }
 
+    /**
+     * Ref: https://developer.twitter.com/en/docs/media/upload-media/api-reference/post-media-upload and
+     * https://developer.twitter.com/en/docs/tweets/post-and-engage/api-reference/post-statuses-update.html "media_ids"
+     */
+    public function publishTweetWithImage(string $status, string $imageFile): void
+    {
+        $media = $this->callPost(self::IMAGE_UPLOAD_URL, [
+            'media' => base64_encode(file_get_contents($imageFile)),
+        ]);
+
+        $this->callPost(self::UPDATE_URL, [
+            'status' => $status,
+            'media_ids' => $media['media_id']
+        ]);
+    }
+
```

Comment:
Ahoj @petrvacha, could you look at this method, **if that's the best way to add image to post?**

It works like a charm: https://twitter.com/VotrubaT/status/917358620068769793

You can look at the rest of this PR as well, but only if you want :) ubt this method is the most important one.